jupyter_resource_usage/static/main.js

```diff
@@ -53,14 +53,16 @@ define([
         $.getJSON({
             url: utils.get_body_data('baseUrl') + 'api/metrics/v1',
             success: function (data) {
-                totalMemoryUsage = humanFileSize(data['rss']);
+                value = data['pss'] || data['rss'];
+                totalMemoryUsage = humanFileSize(value);
 
                 var limits = data['limits'];
                 var display = totalMemoryUsage;
 
                 if (limits['memory']) {
-                    if (limits['memory']['rss']) {
-                        maxMemoryUsage = humanFileSize(limits['memory']['rss']);
+                    limit = limits['memory']['pss'] || limits['memory']['rss'];
```

Comment:
Maybe worth harmonising how this should be coalesced? In `memoryUsage.tsx` `??` is used but there it's `||`

```suggestion
                    limit = limits['memory']['pss'] ?? limits['memory']['rss'];
```

Reply:
Sounds good. Most browsers should now support it by default.